src/components/molecules/AdminVenueCard/AdminVenueCard.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import { AnyVenue } from "types/venues";
+
+import { DEFAULT_VENUE_LOGO } from "settings";
+
+import { WithId } from "utils/id";
+
+import "./AdminVenueCard.scss";
+
+export interface AdminVenueCardProps {
+  venue: WithId<AnyVenue>;
+}
+
+export const AdminVenueCard: React.FC<AdminVenueCardProps> = ({ venue }) => {
+  return (
+    <div className="admin-venue-card">
+      <div
+        className="admin-venue-card__bg"
+        style={{
+          backgroundImage: `url(${venue.mapBackgroundImageUrl})`,
+        }}
+      ></div>
```

Comment:
Since the div has no content, use the 'single tag' format:

```suggestion
      <div
        className="admin-venue-card__bg"
        style={{
          backgroundImage: `url(${venue.mapBackgroundImageUrl})`,
        }}
      />
```